Jenkinsfile

```diff
@@ -42,11 +42,6 @@ pipeline {
         }
 
         stage('Build app Docker images') {
-          environment {
```

Comment:
I'm injecting these via the Dockerfile ENV vars which are available in build, as such they don't need to be defined in the CI.